var/spack/repos/builtin/packages/armpl-gcc/package.py

```diff
@@ -262,3 +261,15 @@ def check_install(self):
         make("-C", armpl_example_dir, "ARMPL_DIR=" + armpl_dir)
         # clean up
         make("-C", armpl_example_dir, "ARMPL_DIR=" + armpl_dir, "clean")
+
+    @run_after("install")
+    def make_pkgconfig_files(self):
+        # ARMpl pkcfonfig files do not have .pc extension and are thus not found by pkg-config
+        armpl_dir = get_armpl_prefix(self.spec)
+        for f in find(join_path(armpl_dir, "pkgconfig"), "*"):
+            copy(f, f + ".pc")
```

Comment:
Is it better to create symlink instead ?

Reply:
symlinks are probably clearer.